### ISSUE TITLE
Are all the fits equally good?

### ISSUE THREAD

**Issue by HWRix** [HWRix/Haloprofile#8]
![screen shot 2015-01-13 at 13 53 45](https://cloud.githubusercontent.com/assets/5234830/5728971/ed99c48e-9b2b-11e4-8564-f72655e1428f.png)

If I look at the two numbers in the blue box, I don't understand them:
the triple power law profile has EXACTLY the same exponents in its 2 outer slopes.
--> It is effectively a singly-broken power law. Why is its likelihood better by 35 (!)
than the best fit singly broken law??

One reason could be the normalization. take e.g. identical r_break=30, alpha1=2.7, alpha2=4.2,
compared to r_break1=30, alpha1=2.7, alpha2=4.2 +  r_break2=50, alpha3=4.2.
Those two cases should give you IDENTICAL likelihoods. Right?
Checke whether they do. If not, check whether they give you the same normalization constant...

**Comment by xuexx** [comment]
You are right. It is the problem of normalisation. When I calculate triple-power-laws, I used 48X48X48 points in MrXDMXFEH, but previously, I always used 20X48X20.

**Comment by HWRix** [comment]
Good. See what happens if you use the same grid for the normalization integral in all cases, and report what comes out
